Extension/src/LanguageServer/client.ts

```diff
@@ -971,6 +971,8 @@ export class DefaultClient implements Client {
         paths.push(localize("installCompiler.string", "Help me install a compiler"));
         paths.push(localize("noConfig.string", "Do not configure a compiler (not recommended)"));
         const index: number = await this.showSelectDefaultCompiler(paths);
+        const action: string = index === -1 ? 'escaped' : index === paths.length - 1 ? 'disable' : index === paths.length - 2 ? 'help' : index === paths.length - 3 ? 'browse' : 'select compiler';
```

Comment:
For readability, can this be replaced with switch/cases?

Reply:
Let's put a note to clean it up for 1.14.3.  I mostly just don't want to jeapordize today's pre-release.